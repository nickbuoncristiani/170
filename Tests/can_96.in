96
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96
16 72 93 75 8 96 77 13 40 81 41 94 67 43 39 19 51 33 61 74 47 15 4 22 9
52
x 1.27226 x 1.40819 1.53042 x 1.46946 1.15158 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.27226 x 1.9779 1.76113 1.79168 1.61803 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.9779 x 1.99851 x 1.97087 x x x x x x x x x x x x x x x x x x x x x x x x 1.21362 1.87348 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.40819 1.76113 1.99851 x 1.45052 1.20179 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.01612 1.60787 1.08307 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.53042 1.79168 x 1.45052 x x 1.30909 1.9747 x x x x x x x x x x x x x x x x x x x x x x x x 1.16013 1.19595 x 1.81295 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.61803 1.97087 1.20179 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.35499 1.37903 1.16474 x 1.84476 x x x x x x x x x x x x 1.03575 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.46946 x x x 1.30909 x x 1.53609 1.26775 1.92554 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.15158 x x x 1.9747 x 1.53609 x 1.39933 1.60765 x x x x x x x x x x x x x x x x x x x x x x x 1.7261 x 1.70482 1.47455 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.26775 1.39933 x 1.48876 1.12235 1.31346 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.92554 1.60765 1.48876 x 1.93374 1.95361 x x x x x x x x x x x x x x x x x x x x x x x 1.09812 1.74851 1.59348 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.12235 1.93374 x 1.37343 1.5286 1.92634 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.31346 1.95361 1.37343 x 1.71238 1.98412 x x x x x x x x x x x x x x x x x x x x x x 1.69369 1.65486 1.82463 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.5286 1.71238 x 1.93461 1.23291 1.79328 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.92634 1.98412 1.93461 x 1.73913 1.57991 x x x x x x x x x x x x x x x x x x x x x 1.50292 1.82284 1.34906 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x 1.23291 1.73913 x 1.90156 1.32053 1.0843 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x 1.79328 1.57991 1.90156 x 1.1282 1.19295 x x x x x x x x x x x x x x x x x x x x 1.8481 1.71578 1.81091 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.32053 1.1282 x 1.04581 1.94022 1.56095 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.0843 1.19295 1.04581 x 1.86705 1.55676 x x x x x x x x x x x x x x x x x x x 1.60779 1.69222 1.40344 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.94022 1.86705 x 1.70026 1.03167 1.35153 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.56095 1.55676 1.70026 x 1.75278 1.66578 x x x x x x x x x x x x x x x x x x 1.68242 1.74076 1.79677 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.03167 1.75278 x 1.60409 1.93255 1.35401 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.35153 1.66578 1.60409 x 1.95763 1.87998 x x x x x x x x x x x x x x x x x 1.70466 1.86724 1.43774 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.93255 1.95763 x 1.27077 1.38098 1.00243 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.35401 1.87998 1.27077 x 1.79002 1.79083 x x x x x x x x x x x x x x x x 1.61828 1.91317 1.98134 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.38098 1.79002 x 1.81352 1.23238 1.42413 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.00243 1.79083 1.81352 x 1.30876 1.92521 x x x x x x x x x x x x x x x 1.15206 1.22875 1.22277 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x 1.23238 1.30876 x 1.00367 1.67329 1.52171 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x 1.42413 1.92521 1.00367 x 1.21616 1.96797 x x x x x x x x x x x x x x 1.252 1.83742 1.40714 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.67329 1.21616 x 1.40134 1.57591 1.24899 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.52171 1.96797 1.40134 x 1.62302 1.57605 x x x x x x x x x x x x x 1.14229 1.91862 1.81911 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.21362 x x 1.35499 x x x x x x x x x x x x x x x x x x x x x x 1.57591 1.62302 x 1.89702 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.87348 x x 1.37903 x x x x x x x x x x x x x x x x x x x x x x 1.24899 1.57605 1.89702 x x x 1.47948 x x x x x x x x x x x 1.60722 1.55651 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.01612 1.16013 1.16474 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.33834 1.93775 x x x x x x x x x x x x x 1.29804 1.34242 1.57179 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.60787 1.19595 x x 1.7261 x x x x x x x x x x x x x x x x x x x x x x x x 1.33834 x x 1.61266 x x x x x x x x x x x x 1.89821 1.76254 x 1.35678 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.08307 x 1.84476 x x x x x x x x x x x x x x x x x x x x x x x x x 1.47948 1.93775 x x x x x x x x x x x x x x 1.30866 1.03021 x 1.17998 x x x x x x x x x x x x 1.61805 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.81295 x x 1.70482 x 1.09812 x x x x x x x x x x x x x x x x x x x x x x x 1.61266 x x 1.09006 x x x x x x x x x x x x 1.65113 x 1.08013 1.57187 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.47455 x 1.74851 x 1.69369 x x x x x x x x x x x x x x x x x x x x x x x 1.09006 x 1.69467 x x x x x x x x x x x x x 1.02261 1.94294 1.03802 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.59348 x 1.65486 x 1.50292 x x x x x x x x x x x x x x x x x x x x x x 1.69467 x 1.84378 x x x x x x x x x x x x x 1.71419 1.24524 1.94658 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.82463 x 1.82284 x 1.8481 x x x x x x x x x x x x x x x x x x x x x 1.84378 x 1.76324 x x x x x x x x x x x x x 1.90927 1.95189 1.55343 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x 1.34906 x 1.71578 x 1.60779 x x x x x x x x x x x x x x x x x x x x 1.76324 x 1.59836 x x x x x x x x x x x x x 1.23339 1.78662 1.46562 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.81091 x 1.69222 x 1.68242 x x x x x x x x x x x x x x x x x x x 1.59836 x 1.1434 x x x x x x x x x x x x x 1.59333 1.14948 1.03319 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.40344 x 1.74076 x 1.70466 x x x x x x x x x x x x x x x x x x 1.1434 x 1.05645 x x x x x x x x x x x x x 1.86802 1.26661 1.89269 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x 1.79677 x 1.86724 x 1.61828 x x x x x x x x x x x x x x x x x 1.05645 x 1.57213 x x x x x x x x x x x x x 1.92769 1.78641 1.67249 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.43774 x 1.91317 x 1.15206 x x x x x x x x x x x x x x x x 1.57213 x 1.38318 x x x x x x x x x x x x x 1.89276 1.81828 1.31146 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.98134 x 1.22875 x 1.252 x x x x x x x x x x x x x x x 1.38318 x 1.97534 x x x x x x x x x x x x x 1.94755 1.33738 1.80731 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.22277 x 1.83742 x 1.14229 x x x x x x x x x x x x x x 1.97534 x 1.06572 x x x x x x x x x x x x x 1.16764 1.78113 1.98674 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.40714 x 1.91862 x 1.60722 x x x x x x x x x x x x x 1.06572 x 1.63217 x x x x x x x x x x x x x 1.2191 1.65128 1.40459 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.03575 x x x x x x x x x x x x x x x x x x x x x x x 1.81911 x 1.55651 x x 1.30866 x x x x x x x x x x x 1.63217 x x x 1.89723 x x x x x x x x x x x 1.20822 1.75346 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.29804 1.89821 1.03021 x x x x x x x x x x x x x x 1.96254 1.78666 x x x x x x x x x x x x x 1.09694 1.83511 1.53297 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.34242 1.76254 x 1.65113 x x x x x x x x x x x x 1.96254 x x 1.86498 x x x x x x x x x x x x 1.63261 1.04443 x 1.6332 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57179 x 1.17998 x x x x x x x x x x x x 1.89723 1.78666 x x x x x x x x x x x x x x 1.82679 1.04372 x 1.99157 x x x x x x x x x x x x 1.72765 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.35678 x 1.08013 1.02261 x x x x x x x x x x x x 1.86498 x x 1.36098 x x x x x x x x x x x x 1.95985 x 1.7074 1.84448 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57187 1.94294 1.71419 x x x x x x x x x x x x x 1.36098 x 1.52387 x x x x x x x x x x x x x 1.03651 1.55627 1.27645 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.03802 1.24524 1.90927 x x x x x x x x x x x x x 1.52387 x 1.30934 x x x x x x x x x x x x x 1.62742 1.34941 1.65687 x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.94658 1.95189 1.23339 x x x x x x x x x x x x x 1.30934 x 1.81192 x x x x x x x x x x x x x 1.58381 1.79785 1.11632 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.55343 1.78662 1.59333 x x x x x x x x x x x x x 1.81192 x 1.26103 x x x x x x x x x x x x x 1.43175 1.68558 1.04464 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.46562 1.14948 1.86802 x x x x x x x x x x x x x 1.26103 x 1.78634 x x x x x x x x x x x x x 1.41874 1.88457 1.28675 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.03319 1.26661 1.92769 x x x x x x x x x x x x x 1.78634 x 1.87454 x x x x x x x x x x x x x 1.49933 1.45383 1.67624 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.89269 1.78641 1.89276 x x x x x x x x x x x x x 1.87454 x 1.48983 x x x x x x x x x x x x x 1.49418 1.25179 1.21652 x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.67249 1.81828 1.94755 x x x x x x x x x x x x x 1.48983 x 1.00427 x x x x x x x x x x x x x 1.01684 1.81469 1.69734 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.31146 1.33738 1.16764 x x x x x x x x x x x x x 1.00427 x 1.56853 x x x x x x x x x x x x x 1.81097 1.54453 1.38081 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.80731 1.78113 1.2191 x x x x x x x x x x x x x 1.56853 x 1.59605 x x x x x x x x x x x x x 1.92611 1.33516 1.12093 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.98674 1.65128 1.20822 x x x x x x x x x x x x x 1.59605 x 1.45878 x x x x x x x x x x x x x 1.53102 1.05431 1.36555 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.61805 x x x x x x x x x x x 1.40459 1.75346 x x 1.82679 x x x x x x x x x x x 1.45878 x x x 1.69091 x x x x x x x x x x x 1.52847 1.99331 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.09694 1.63261 1.04372 x x x x x x x x x x x x x x 1.27136 1.27845 x x x x x x x x x x x x x 1.60423 1.83822 1.59214 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83511 1.04443 x 1.95985 x x x x x x x x x x x x 1.27136 x x 1.22087 x x x x x x x x x x x x 1.76665 1.73811 x 1.34326 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.53297 x 1.99157 x x x x x x x x x x x x 1.69091 1.27845 x x x x x x x x x x x x x x 1.39041 1.55525 x 1.41549 x x x x x x x x x x x x 1.5134
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.6332 x 1.7074 1.03651 x x x x x x x x x x x x 1.22087 x x 1.28472 x x x x x x x x x x x x 1.35503 x 1.77895 1.21495 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.84448 1.55627 1.62742 x x x x x x x x x x x x x 1.28472 x 1.2014 x x x x x x x x x x x x x 1.99899 1.66576 1.50814 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.27645 1.34941 1.58381 x x x x x x x x x x x x x 1.2014 x 1.0941 x x x x x x x x x x x x x 1.0601 1.18403 1.05183 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.65687 1.79785 1.43175 x x x x x x x x x x x x x 1.0941 x 1.11391 x x x x x x x x x x x x x 1.96543 1.06396 1.29739 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.11632 1.68558 1.41874 x x x x x x x x x x x x x 1.11391 x 1.45232 x x x x x x x x x x x x x 1.6901 1.10273 1.50325 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.04464 1.88457 1.49933 x x x x x x x x x x x x x 1.45232 x 1.27913 x x x x x x x x x x x x x 1.33987 1.3364 1.46227 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.28675 1.45383 1.49418 x x x x x x x x x x x x x 1.27913 x 1.41031 x x x x x x x x x x x x x 1.44145 1.23866 1.22326 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.67624 1.25179 1.01684 x x x x x x x x x x x x x 1.41031 x 1.08165 x x x x x x x x x x x x x 1.50197 1.52687 1.78764 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.21652 1.81469 1.81097 x x x x x x x x x x x x x 1.08165 x 1.16107 x x x x x x x x x x x x x 1.70814 1.93518 1.18892 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.69734 1.54453 1.92611 x x x x x x x x x x x x x 1.16107 x 1.92152 x x x x x x x x x x x x x 1.29013 1.20984 1.1151 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.38081 1.33516 1.53102 x x x x x x x x x x x x x 1.92152 x 1.35291 x x x x x x x x x x x x x 1.67194 1.69836 1.9576 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.12093 1.05431 1.52847 x x x x x x x x x x x x x 1.35291 x 1.77126 x x x x x x x x x x x x x 1.96033 1.97739 1.35437
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.72765 x x x x x x x x x x x 1.36555 1.99331 x x 1.39041 x x x x x x x x x x x 1.77126 x x x 1.24478 x x x x x x x x x x x 1.20438 1.09306
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.60423 1.76665 1.55525 x x x x x x x x x x x x x x 1.64546 1.37143 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83822 1.73811 x 1.35503 x x x x x x x x x x x x 1.64546 x x 1.52586 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.59214 x 1.41549 x x x x x x x x x x x x 1.24478 1.37143 x x x x x x x x x x x x x x 1.97492
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.34326 x 1.77895 1.99899 x x x x x x x x x x x x 1.52586 x x 1.40409 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.21495 1.66576 1.0601 x x x x x x x x x x x x x 1.40409 x 1.30146 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50814 1.18403 1.96543 x x x x x x x x x x x x x 1.30146 x 1.48731 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.05183 1.06396 1.6901 x x x x x x x x x x x x x 1.48731 x 1.1849 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.29739 1.10273 1.33987 x x x x x x x x x x x x x 1.1849 x 1.54596 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50325 1.3364 1.44145 x x x x x x x x x x x x x 1.54596 x 1.66611 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.46227 1.23866 1.50197 x x x x x x x x x x x x x 1.66611 x 1.83767 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.22326 1.52687 1.70814 x x x x x x x x x x x x x 1.83767 x 1.4559 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.78764 1.93518 1.29013 x x x x x x x x x x x x x 1.4559 x 1.06191 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.18892 1.20984 1.67194 x x x x x x x x x x x x x 1.06191 x 1.87586 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.1151 1.69836 1.96033 x x x x x x x x x x x x x 1.87586 x 1.66294 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.9576 1.97739 1.20438 x x x x x x x x x x x x x 1.66294 x 1.00579
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.5134 x x x x x x x x x x x 1.35437 1.09306 x x 1.97492 x x x x x x x x x x x 1.00579 x
