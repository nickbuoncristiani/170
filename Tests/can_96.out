52 37 12 13 15 17 20 43 60 77 60 59 74 75 58 41 18 19 21 24 45 62 79 96 67 49 33 5 8 9 11 13 15 17 19 22 42 43 20 17 16 39 40 14 13 11 9 8 34 49 67 64 47 62 77 61 46 30 31 3 4 34 52
21
52 81
13 13
15 15
43 43
77 77 94 93
74 74
75 75
41 41
19 19
96 96
67 67
33 33 51 4
8 8
9 9
22 22
42 72
16 16
39 39
40 40
47 47
61 61
